package.json

```diff
@@ -93,10 +93,10 @@
     "eth-block-tracker": "^4.1.0",
     "eth-contract-metadata": "github:MetaMask/eth-contract-metadata#dc68506221859bc90792bc5e0279a6835f2484d8",
     "eth-ens-namehash": "^2.0.8",
-    "eth-hd-keyring": "^1.2.2",
+    "eth-hd-keyring": "github:brave-experiments/eth-hd-keyring#staging",
     "eth-json-rpc-filters": "^3.0.4",
     "eth-json-rpc-infura": "^3.2.0",
-    "eth-keyring-controller": "^3.3.1",
+    "eth-keyring-controller": "github:brave-experiments/KeyringController",
```

Comment:
this repo also needs an update once the eth-hd-keyring PR is merged (to move it from brave-experiments to brave)